deegree-core/deegree-core-base/src/main/java/org/deegree/geojson/GeoJsonWriter.java

```diff
@@ -219,7 +281,10 @@ else if (propertyType instanceof FeaturePropertyType) {
 			exportFeaturePropertyType(property);
 		}
 		else if (propertyType instanceof GeometryPropertyType) {
-			// Do nothing as geometry was exported before.
+			// geometry was exported before
+			// need to throw an exception here, because if we do nothing the Json is
+			// invalid (condition should be handled before)
+			throw new IOException("Geometry should not be rendered as part of GeoJson properties.");
```

Comment:
Can you please explain why do you see the need to throw an exception here or is it intended to prevent misuse of the API here? Does throwing the exception stops the generation of the GFI GeoJSON output?

Reply:
This code should not be reached, the exception is thrown to raise a failure if it does happen nonetheless.

If such a case occurs and the behavior here would be as before, doing nothing, no valid Json would be created and the process would fail with the "Dangling name" exception mentioned in the PR description, because there would be a Json field w/o a value.

So the behavior would be similar, just a different exception.

Now we catch that case earlier though to prevent the field from being written at all and thus to prevent the exception.

The intention here seemed clearly to not export the geometry again, so I guess only other alternative would be creating a `null` value for the field or an empty object. Depends on what is preferable - I went for omitting the field in that case.